doc/user_guide/statements/insert.md

```diff
@@ -0,0 +1,71 @@
+# INSERT 
+
+You can construct [`INSERT`](https://docs.exasol.com/sql/insert.htm) SQL statements using the `Insert` class.
+
+## Creating `INSERT` Statements
+
+Create an `INSERT` statement:
+
+```java
+        final Insert insert = StatementFactory.getInstance()
+                .insertInto("tableName")
+                .values("value1","value2","value3");
+```
+
+Create an `INSERT` statement for specific fields:
+
+```java
+        final Insert insert = StatementFactory.getInstance()
```

Comment:
Just an idea: How about adding the rendered statement to each example.

Reply:
Not a bad idea per se, but then we should do this for all of the documentation and this would be outside of the scope of this ticket.